dsl/pace/dsl/gt4py_utils.py

```diff
@@ -401,22 +419,16 @@ def index(array, key):
 
 
 def moveaxis(array, source: int, destination: int):
```

Comment:
Not for this PR, but we should probably try to delete these functions, in favor of e.g. `quantity.np.moveaxis`.

Reply:
Good idea, I added an issue [here](https://github.com/ai2cm/pace/issues/383).